tianshou/policy/base.py

```diff
@@ -556,6 +556,9 @@ def compute_episodic_return(
         advantage + value, which is exactly equivalent to using :math:`TD(\lambda)`
         for estimating returns.
 
+        Setting v_s_ and v_s to None (or all zeros) and gae_lambda to 1.0 calculates the
```

Comment:
@MischaPanch @bordeauxred this has broken the docs build on master. Must add quotes to symbols.

Reply:
I was really confused by the build failing on my branch - and it was because of changes that aren't even in my branch.

Reply:
Thanks for spotting. Fixed in #1132